app/javascript/dashboard/components/widgets/conversation/linear/index.vue

```diff
@@ -80,17 +88,28 @@ const loadLinkedIssue = async () => {
     const issues = response.data;
     linkedIssue.value = issues && issues.length ? issues[0] : null;
   } catch (error) {
-    useAlert(error?.message || t('INTEGRATION_SETTINGS.LINEAR.LOADING_ERROR'));
+    const errorData = error.response.data;
+    const errorMessage =
+      errorData?.error?.errors?.[0]?.message ||
+      t('INTEGRATION_SETTINGS.LINEAR.LOADING_ERROR');
+    useAlert(errorMessage);
```

Comment:
Please take a look at this [comment](https://github.com/chatwoot/chatwoot/pull/9538#discussion_r1616654267).